### PR TITLE
Added browserextension.dev to News

### PR DESCRIPTION
**Pull request by lindaicing**
Added browserextension.dev to News

Adjusted styling and spacing. Added spaces in CSS for formatting. 
Updated article heading sizes of articles:
Screen Shot 2020-07-24 at 8 24 35 PM

News Section is still hidden, but when not hidden it looks like this:
Screen Shot 2020-07-24 at 8 22 57 PM

### CODE REVIEW

File: Website-and-Backend/src/main/resources/public/index.html [hunk 1]
```diff
@@ -114,9 +114,9 @@ <h3>About Ratings</h3>
             <h2><span class="colorCream">Ethicli</span> <span class="colorLGreen">in the News</span></h2>
             <section>
                 <article>
-                    <h3>Coming Soon</h3>
-                    <p>News about Ethicli coming soon!</p>
-                    <p class="readmore"><a>Read More</a></p>
+                    <h3>BrowserExtension.dev</h3>
+                    <p>We've been featured by <a href="https://browserextension.dev" target="_blank">BrowserExtension.dev</a>, which is a website that discusses and explores new web extensions.</p>
```
Comment (CubeOfCheese) [inline Website-and-Backend/src/main/resources/public/index.html:118]: You should link to our specific feature.

Comment (CubeOfCheese) [reply Website-and-Backend/src/main/resources/public/index.html:118]: https://browserextension.dev/blog/ethicli/

Comment (CubeOfCheese) [reply Website-and-Backend/src/main/resources/public/index.html:118]: Oh wait my bad nevermind

**Comment by lindaicing** [comment]
> Maybe the other two should just be hidden. And then we could actually merge this and have it on our website. What do you think?
> Also, the menu color changing needs to be changed with this section being added back in

Ah shoot totally forgot about the menu, thank you!

**Comment by lindaicing** [comment]
Ethicli in the News - Now shown
Screen Shot 2020-07-24 at 11 23 49 PM

Featured Articles - Updated most recent article
Screen Shot 2020-07-24 at 11 24 12 PM

Updated About Section to the phrase we came up with
Screen Shot 2020-07-24 at 11 25 16 PM

Updated FAQ intro - Linked Intro Medium Article
Screen Shot 2020-07-24 at 11 26 07 PM

**Pull request merged**